plugins/schedule/base/lib/api/filters/KalturaScheduleEventFilter.php

```diff
@@ -13,6 +13,12 @@ class KalturaScheduleEventFilter extends KalturaScheduleEventBaseFilter
 		"parentResourceIdsLike" => "_like_parent_resource_ids",
 		"parentResourceIdsMultiLikeOr" => "_mlikeor_parent_resource_ids",
 		"parentResourceIdsMultiLikeAnd" => "_mlikeand_parent_resource_ids",
+		"templateEntryCategoriesIdsLike" => "_like_template_entry_categories_ids",
+		"templateEntryCategoriesIdsMultiLikeAnd" => "_mlikeand_template_entry_categories_ids",
+		"templateEntryCategoriesIdsMultiLikeOr" => "_mlikeor_template_entry_categories_ids",
+		"systemNamesLike" => "_like_system_names",
```

Comment:
Please rename to resourceSystemNamesLike, resourceSystemNamesMultiLikeOr and resourceSystemNamesMultiLikeAnd.